BrowserKit/Sources/SiteImageView/ImageProcessing/ImageHandler.swift

```diff
@@ -67,6 +67,20 @@ class DefaultImageHandler: ImageHandler {
             return await fetchFaviconFromFetcher(imageModel: imageModel)
         }
     }
+    
+    private func loadDefaultFaviconFromBundle(assetName: String) throws -> UIImage {
+        do {
+            return try getBundleImage(assetName: assetName)
+        } catch {
+            logger.log(
+                "Could not get image from bundle",
+                level: .warning,
+                category: .images,
+                extra: ["assetName": assetName]
+            )
+            throw SiteImageError.noImageInBundle
+        }
+    }
```

Comment:
Yesss this is great, this should just give us the log for assets we EXPECT must be in the bundle. 😁  👍

Reply:
Thank you very much @ih-codes